src/main/java/eiteam/esteemedinnovation/boiler/ContainerBoiler.java

```diff
@@ -111,8 +112,8 @@ public void updateProgressBar(int id, int data) {
     }
 
     @Override
-    public boolean canInteractWith(@Nonnull EntityPlayer par1EntityPlayer) {
-        return tileEntity.isUsableByPlayer(par1EntityPlayer);
+    public boolean canInteractWith(EntityPlayer playerIn) {
+        return true;
```

Comment:
Why was this changed?

Reply:
isUsableByPlayer is a method from IInventory. Since we are no longer using it, this method does not exist anymore.